Makefile.core.mk

```diff
@@ -634,7 +634,7 @@ $(HELM): $(ISTIO_OUT)
 	bin/init_helm.sh
 
 $(HOME)/.helm:
-	$(HELM) init --client-only
+	$(HELM) init --stable-repo-url https://charts.helm.sh/stable --client-only
```

Comment:
was this change necessary to run locally or on prow?

Reply:
It was necessary to run either locally or on prow. Helm killed the built-in default URL for charts after they ended support for Helm 2. This is an archive URL that Helm now suggests people use.

Reply:
This is the error that was seen in prow:

```
test-317de0adb52549ea/_suite_context/istio-deployment-173388722/helm/repository/repositories.yaml \nAdding stable repo with URL: https://kubernetes-charts.storage.googleapis.com \nError: error initializing: Looks like \"https://kubernetes-charts.storage.googleapis.com\" is not a valid chart repository or cannot be reached: Failed to fetch https://kubernetes-charts.storage.googleapis.com/index.yaml : 403 Forbidden\n"
```

Reply:
ah, thanks. so it's not related to a change in version on our side, but they killed the server. gotcha